scripts/docker/requirements.txt

```diff
@@ -0,0 +1,27 @@
+apt-utils
+dialog
+software-properties-common=0.99.9.12
+build-essential=12.8ubuntu1.1
+libgoogle-glog-dev=0.4.0-1build1
+libyaml-cpp-dev=0.6.2-4ubuntu1
+git
+git-lfs
+clang-6.0=1:6.0.1-14
+libboost-all-dev=1.71.0.0ubuntu2
+libsndfile1=1.0.28-7ubuntu0.2
+pandoc
+libtbb-dev
+libcapstone-dev
+pkg-config
+cmake=3.16.3-1ubuntu1.20.04.1
+curl
+wget
+python3-pip
+libhwloc-dev
+
+libhdf5-serial-dev
+ruby=1:2.7+1
+python3.8-venv=3.8.10-0ubuntu1~20.04.9
+
+cargo
+ninja-build
```

Comment:
new line

Reply:
Also damn, I'm wondering if we can capture these requirements in one nice file somewhere rather than copy pasta in

- miw playbooks
- dev deps JSONs in `.github/actions`
- README

probably some other places